app/views/step_by_step_pages/show.html.erb

```diff
@@ -0,0 +1,55 @@
+<%
+  links = [
+    {
+      text: 'Step by step publisher',
+      href: step_by_step_pages_path
+    },
+    {
+      text: 'Edit content'
+    }
+  ]
+%>
+<%= render 'partials/step_breadcrumb', links: links %>
+
+<%= render 'partials/form_notice', message: notice, status: "success" %>
+
+<h1><%= @step_by_step_page.title %></h1>
+
+<%= @step_by_step_page.introduction %>
+
+<p>Meta description: <%= @step_by_step_page.description %></p>
+
+<p class="text-right">
+  <%= link_to("Preview", step_by_step_page_preview_path(@step_by_step_page), class: "btn btn-info") %>
```

Comment:
I wonder if we should even remove the preview? The current implementation is only very basic and has non-functional elements as well (mainly the "share preview link" link).